<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>29</width>
    <height>29</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_image">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>341</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_connect">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>67</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="form_connect">
       <item row="0" column="0">
        <widget class="QLabel" name="lb_serial">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Porta Serial:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_velocidade">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Velocidade:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbox_serial"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbox_veloc"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pb_conectar">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Conectar</string>
      </property>
      <property name="icon">
       <iconset>
        <selectedon>:/resource/iic.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>21</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_reiniciar">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reiniciar</string>
      </property>
      <property name="icon">
       <iconset>
        <selectedon>:/resource/iic2.png</selectedon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>22</width>
        <height>21</height>
       </size>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_temp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>251</width>
        <height>91</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_temp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Temperatura da cerveja (Â°C)</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_status">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string>Desconectado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lb_logo">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>141</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icones.qrc">:/resource/ThermoBeercomF.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_cadastro">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>311</width>
        <height>147</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbcd_nome">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nome:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ledit_nome"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbcd_tipo">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tipo:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ledit_tipo"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbcd_link">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Link da receita:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="ledit_link"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Levedura:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="lev_combo">
         <item>
          <property name="text">
           <string>Escolha uma levedura...</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Cadastro das cervejas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="dedit_inicio">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>79</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_iniferm">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>InÃ­cio da FermentaÃ§Ã£o</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lb_termferm">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TÃ©rmino da FermentaÃ§Ã£o</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDateEdit" name="dedit_termino">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>79</width>
        <height>18</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_qtd">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quantidade Produzida</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spbox_qtd">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>180</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar cerveja</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icones.qrc"/>
 </resources>
 <connections/>
</ui>
